job-service-acceptance-tests/src/integration-test/java/com/hpe/caf/jobservice/acceptance/JobServiceDatabaseUtil.java

```diff
@@ -50,6 +50,42 @@ public static void assertJobStatus(final String jobId, final String expectedStat
         }
     }
 
+    public static int getJobDelay(final String jobId) throws SQLException
+    {
+        try (final Connection dbConnection = getDbConnection()) {
+
+            int delay = 0;
```

Comment:
I don't see any point in initialising this.  Just declare it with `final int` at the point that the value is returned.  Same goes for `eligibleRunDate` in the next function.